checkov/sast/engines/prisma_engine.py

```diff
@@ -190,13 +196,36 @@ def run_go_library(self, languages: Set[SastLanguages],
         # convert our byte array to a string
         analyze_code_string = analyze_code_bytes.decode('utf-8')
         d = json.loads(analyze_code_string)
+
         try:
             result = PrismaReport(**d)
         except ValidationError as e:
             result = create_empty_report(list(languages))
             result.errors = {REPORT_PARSING_ERRORS: [str(err) for err in e.errors()]}
         return self.create_report(result)
 
+    def run_go_library_list_policies(self, document: Dict[str, Any]) -> SastPolicies:
+        library = ctypes.cdll.LoadLibrary(self.lib_path)
+        list_policies = library.listPolicies
+        list_policies.restype = ctypes.c_void_p
+
+        # send the document as a byte array of json format
+        list_policies_output = list_policies(json.dumps(document).encode('utf-8'))
+
+        # we dereference the pointer to a byte array
+        list_policies_bytes = ctypes.string_at(list_policies_output)
+
+        # convert our byte array to a string
+        list_policies_string = list_policies_bytes.decode('utf-8')
+        d = json.loads(list_policies_string)
```

Comment:
let's wrap this in `try..except` clause as well, and catch JSON dumps/loads and ctypes related exceptions